### PR TITLE
EOS-13759 [LDR-R1 SW Upgrade] auth server side

### PR DESCRIPTION
**Pull request by shalakadharap**

### CODE REVIEW

File: auth/resources/authserver.properties [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
-
+version=1
```
Comment (abhilekh268) [inline auth/resources/authserver.properties:19]: make it 1.0.0..as we do in rpms.

Comment (shalakadharap) [reply auth/resources/authserver.properties:19]: as discussed, keeping it 1

**Review by abhilekh268 — APPROVED** [review]

**Comment by shalakadharap** [comment]
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server/43/
Jenkins build success

**Review by shanbhagsv1 — APPROVED** [review]
Looks good for me

**Pull request merged**